tests/end2end-tests/data_process/test.sh

```diff
@@ -447,6 +452,13 @@ cp -r /tmp/em_remap/partial-emb/n0/*1.pt /tmp/em_remap/partial-emb/1/n0/
 cp -r /tmp/em_remap/partial-emb/n1/*0.pt /tmp/em_remap/partial-emb/0/n1/
 cp -r /tmp/em_remap/partial-emb/n1/*1.pt /tmp/em_remap/partial-emb/1/n1/
 
+# Add relation2id_map.json and into /tmp/em_remap/partial-emb/
+# remap_result should ignore this file
+echo 'Dummy' > /tmp/em_remap/partial-emb/0/rel_emb.pt
+echo 'Dummy' > /tmp/em_remap/partial-emb/1/rel_emb.pt
+echo 'Dummy' > /tmp/em_remap/partial-emb/0/rel_emb.pt
+echo 'Dummy' > /tmp/em_remap/partial-emb/1/rel_emb.pt
```

Comment:
Lines 1-3 and 2-4 are identical and simply replace the contents of the existing file. We can use
```suggestion
for i in 0 1; do
  touch /tmp/em_remap/partial-emb/$i/rel_emb.pt
done
```